blog/_posts/rebecca.md

```diff
@@ -15,7 +15,7 @@ Today we want to show you, who's behind that. Who made RGSoC look so amazing and
 <img src="https://f.cloud.github.com/assets/1711357/1329775/0904b3de-351e-11e3-9ee3-51dc00c26c75.jpg" width="300" style="padding-right: 2px;">
 <img src="https://f.cloud.github.com/assets/1711357/1329736/2cd0fd96-351d-11e3-8efb-03b809ebee37.jpg" height="533" style="padding-left: 2px;">
 
-Some weeks ago we met at the Travis CI office and - some days now it's hard to believe - it was really warm outside. We chatted and I asked her some questions to get to know the woman whith the most amazing tatoos and the one who can draw like I always dreamed of. After doing a lot of funny friday-hug-pics, we settled down on the couch and started talking. 
+Some weeks ago we met at the Travis CI office and - some days now it's hard to believe - it was really warm outside. We chatted and I asked her some questions to get to know the woman whith the most amazing tatoos and the one who can draw like I always dreamed of. After doing a lot of funny friday-hug-pics, we settled down on the couch and started talking.
```

Comment:
could you revert all the changes in here as well / remove this file from the pull request? :+1: